### PR TITLE
FutureSight.Bingo version 1.3.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103042)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230420233358

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 5:48:51 PM: 2023-04-20 17:47:40.122 [CLI ] Package hash verification failed. SHA256 in manifest [9e581d70f5cec3bbc40d89fcbda14e7f83c11e3c7229273172b254602612b625] does not match download [3022a67931f09ffdd75d35000d19bae9885190d00c0a51b51621c7b06b97687f]

File: manifests/f/FutureSight/Bingo/1.3.0/FutureSight.Bingo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: FutureSight.Bingo
+PackageVersion: 1.3.0
+Installers:
+- InstallerUrl: https://download1640.mediafire.com/oz35hmahfg8gJckIbNivpo7RHfYSu7nT6oPonwUVK26yJUEUvpcd55rNtkFifr4kmK8FCbNA9do0_NX8gVF8DRmB0LALjk8/eaq7ak43r3pku5k/Bingo_1-3-0.zip"           id="downloadButton
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Bingo_1-3-0\Bingo.exe
+  InstallerSha256: 9E581D70F5CEC3BBC40D89FCBDA14E7F83C11E3C7229273172B254602612B625
```
Comment (stephengillie) [inline manifests/f/FutureSight/Bingo/1.3.0/FutureSight.Bingo.installer.yaml:13]:
```suggestion
  InstallerSha256: 3022A67931F09FFDD75D35000D19BAE9885190D00C0A51B51621C7B06B97687F
```

**Comment by upintheairsheep** [comment]
The creator only put their item on mediafire

File: manifests/f/FutureSight/Bingo/1.3.0/FutureSight.Bingo.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: FutureSight.Bingo
+PackageVersion: 1.3.0
+Installers:
+- InstallerUrl: https://download1640.mediafire.com/oz35hmahfg8gJckIbNivpo7RHfYSu7nT6oPonwUVK26yJUEUvpcd55rNtkFifr4kmK8FCbNA9do0_NX8gVF8DRmB0LALjk8/eaq7ak43r3pku5k/Bingo_1-3-0.zip"           id="downloadButton
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: Bingo_1-3-0\Bingo.exe
+  InstallerSha256: 3022A67931F09FFDD75D35000D19BAE9885190D00C0A51B51621C7B06B97687F
```
Comment (stephengillie) [inline manifests/f/FutureSight/Bingo/1.3.0/FutureSight.Bingo.installer.yaml:13]:
```suggestion
```suggestion
- InstallerUrl: https://download1640.mediafire.com/zd3zyp82cufg2SUBmXrsYHA6tE32pkoypg4jsmrQttAnlrv0TI8-9ZnY1FzJS4-pYUizT9cIOPreC8--swFLbi9Vkwmypi3AxWrTC8__VGSAqbJBdLkUruCj_ZAU35Po7eBELHiapqqVHMRwEz6rzHzW9O3Bxcf9oA2AVctgqp2U/eaq7ak43r3pku5k/Bingo_1-3-0.zip
  Architecture: x86
  InstallerType: zip
  NestedInstallerType: exe
  NestedInstallerFiles:
  - RelativeFilePath: Bingo_1-3-0\Bingo.exe
  InstallerSha256: 9E581D70F5CEC3BBC40D89FCBDA14E7F83C11E3C7229273172B254602612B625
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Manifest is invalid. (Line: 7, Col: 1, Idx: 206) - (Line: 7, Col: 2, Idx: 207): While scanning for the next token, found character that cannot start any token.

File: manifests/f/FutureSight/Bingo/1.3.0/FutureSight.Bingo.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FutureSight.Bingo
+PackageVersion: 1.3.0
+Installers:
+```suggestion
```
Comment (stephengillie) [inline manifests/f/FutureSight/Bingo/1.3.0/FutureSight.Bingo.installer.yaml:7]:
```suggestion
```

**Comment by upintheairsheep** [comment]
Hello

**Comment by stephengillie** [comment]
> The creator only put their item on mediafire

Sorry for the delay. Is this a nightly package that receives frequent updates? The hash seems to change faster than we can validate. Also, the InstallerUrl seems to have a somewhat-random section. Mediafire might not be an ideal host for downloads through the WinGet package manager.

**Comment by upintheairsheep** [comment]
It's a non-static URL. There is a discussion to allow the use of external
sources for this stuff.

**Comment by upintheairsheep** [comment]
https://github.com/microsoft/winget-pkgs/discussions/97158

**Comment by ItzLevvie** [comment]
[Policy] Area-External

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by stephengillie** [comment]
Closing PR as stale.

**Pull request closed without merging**